vue.config.js

```diff
@@ -0,0 +1,108 @@
+const path = require(`path`)
+const fs = require(`fs`)
+const webpack = require(`webpack`)
+const CSPWebpackPlugin = require(`csp-webpack-plugin`)
+
+function resolve(dir) {
+  return path.join(__dirname, dir)
+}
+
+const commitHash = require(`child_process`)
+  .execSync(`git rev-parse HEAD`)
+  .toString()
+  .trim()
+
+module.exports = {
+  configureWebpack: () => {
+    const config = {
+      resolve: {
+        alias: {
+          src: resolve(`src`),
+          "@": resolve(`src`),
+          assets: resolve(`src/assets`),
+          scripts: resolve(`src/scripts`),
+          common: resolve(`src/components/common`),
+          transactions: resolve(`src/components/transactions`),
+          govern: resolve(`src/components/govern`),
+          staking: resolve(`src/components/staking`),
+          wallet: resolve(`src/components/wallet`),
+          test: resolve(`test`)
+        },
+        extensions: [`.js`, `.vue`, `.css`]
+      },
+      plugins: [
+        new webpack.DefinePlugin({
+          "process.env": {
+            NODE_ENV: JSON.stringify(process.env.NODE_ENV),
+            RPC: JSON.stringify(process.env.RPC),
+            STARGATE: JSON.stringify(process.env.STARGATE),
+            RELEASE: JSON.stringify(commitHash),
+            GOOGLE_ANALYTICS_UID: JSON.stringify(
+              process.env.GOOGLE_ANALYTICS_UID
+            ),
+            SENTRY_DSN: JSON.stringify(process.env.SENTRY_DSN)
```

Comment:
can be removed?

Reply:
removing sentry in another PR. would mean a lot more work.